katcp/test/test_fake_clients.py

```diff
@@ -51,6 +57,66 @@ def test_sensors(self):
         self.assert_sensor_equal_description(a_string, dict(
```

Comment:
Since the new assertions use literal dicts (to make the linter happy), suggest the old two test do the same for consistency.